### PR TITLE
Add better description handler for cucumber features

### LINKED ISSUE #740: [Allure & Cucumber-JS] There is no description in the Allure Report
**Describe the bug**
All lines in feature files that come after Feature, Scenario or Scenario Outline are not displayed in the Allure report.

allure-cucumberjs: 2.4.0
@cucumber/cucumber: 9.3.0

**To Reproduce**
1. Prepare a feature file with approximately the following content
```
Feature: Some feature
  Feature's description
  
  Scenario: Some scenario
    Description of scenario

    Given some give step
    When some when step
    Then some than step
```
2. Implement the steps 
3. Run the test
4. Generate the report

**Expected behavior**
The "Description" block is expected, by analogy with how it is done in Java
image

**Actual behavior**
There is no "Description" block
image

### PR DESCRIPTION
**Pull request by epszaw**
### Context

fixes #740 

#### Checklist
- [x] [Sign Allure CLA][cla]
- [x] Provide unit tests

[cla]: https://cla-assistant.io/accept/allure-framework/allure2

### CODE REVIEW

File: packages/allure-cucumberjs/src/CucumberJSAllureReporter.ts [hunk 1]
```diff
@@ -299,13 +299,18 @@ export class CucumberJSAllureFormatter extends Formatter {
     const currentTest = new AllureTest(this.allureRuntime, Date.now());
     const thread = data.workerId || ALLURE_THREAD_NAME || process.pid.toString();
     const fullName = `${pickle.uri}#${pickle.name}`;
+    // sometimes description starts with "Description:", but it's not actually required
+    const description = scenario?.description
+      ? scenario.description.replace(/\s+Description:\s?/, "")
+      : doc?.feature?.description;
```
Comment (baev) [inline packages/allure-cucumberjs/src/CucumberJSAllureReporter.ts:305]: Maybe always join the feature description with the scenario description?

File: packages/allure-cucumberjs/src/CucumberJSAllureReporter.ts [hunk 2]
```diff
@@ -299,13 +299,18 @@ export class CucumberJSAllureFormatter extends Formatter {
     const currentTest = new AllureTest(this.allureRuntime, Date.now());
     const thread = data.workerId || ALLURE_THREAD_NAME || process.pid.toString();
     const fullName = `${pickle.uri}#${pickle.name}`;
+    // sometimes description starts with "Description:", but it's not actually required
```
Comment (baev) [inline packages/allure-cucumberjs/src/CucumberJSAllureReporter.ts:302]: Cucumber supports i18n. Is it always `Description:` or depends on the used language?

**Review by baev — CHANGES_REQUESTED** [review]

File: packages/allure-cucumberjs/src/CucumberJSAllureReporter.ts (hunk 2 above)
Comment (epszaw) [reply packages/allure-cucumberjs/src/CucumberJSAllureReporter.ts:302]: True, it doesn't relates on it at all, so we don't need to transform the description

File: packages/allure-cucumberjs/src/CucumberJSAllureReporter.ts (hunk 1 above)
Comment (epszaw) [reply packages/allure-cucumberjs/src/CucumberJSAllureReporter.ts:305]: I afraid it'll produce strange descriptions with a lot of duplicated data. Guess, we should entirely replace feature description by a local one

File: packages/allure-cucumberjs/src/CucumberJSAllureReporter.ts (hunk 2 above)
Comment (epszaw) [reply packages/allure-cucumberjs/src/CucumberJSAllureReporter.ts:302]: Fixed and pushed

**Review by baev — APPROVED** [review]

**Pull request merged**